lib/generators/solidus_stripe/install/templates/app/javascript/controllers/solidus_stripe_intent_controller.js

```diff
@@ -32,6 +32,9 @@ export default class extends Controller {
 
   // @action
   async handleSubmit(e) {
+    // Bail out if not on the payment method form.
+    if (e.target !== this.radioOutletElement.form) return
```

Comment:
How could that happen??

Reply:
If you have any other form in the page that triggers a submit event (e.g. a subscribe to newsletter in the footer) this action will be triggered, since we listen for a generic `submit@window`. When we'll update the starter frontend with stimulus controllers and can be used as [outlets](https://stimulus.hotwired.dev/reference/outlets) we might be able to clean up this considerably.